extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/multipart/MultipartResource.java

```diff
@@ -67,4 +67,18 @@ public String sameName(FormDataSameFileName formData) {
                 + formData.xmlFiles.size();
     }
 
+    @POST
+    @Produces(MediaType.TEXT_PLAIN)
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @Path("/optional")
+    @NonBlocking
+    public String simple(@MultipartForm FormData formData, Integer times) {
```

Comment:
Are you sure this will compile? I propose remaning to `optional`

Reply:
sure... thanks again